.env.sample

```diff
@@ -3,9 +3,15 @@ BMS_RELEASE=16.0
 BMS_DB_HOST=bmsmysql
 BMS_DB_PASS=CHANGE_PASSWORD
 BMS_CROPS=maize
+BMS_JAVA_OPTS=-server -Xms2g -Xmx2g -XX:+CMSClassUnloadingEnabled -Dspring.profiles.active=development
```

Comment:
Let's have 2 variables: `BMS_MEMORY` and `LIQUIBASE_PARAM` and build JAVA_OPTS from there. `LIQUIBASE_PARAM= -Dspring.profiles.active=development` by default and put a comment above the property it to keep it blank in order to disable